src/js/shield_defs.js

```diff
@@ -2751,10 +2765,7 @@ export function loadShields() {
       "US:TX:Loop:Express:Toll": ["EXPR", "LOOP"],
     },
   };
-  shields["US:TX:Toll"] = shields["US:TX:NTTA"] = roundedRectShield(
```

Comment:
This eliminates support for `US:TX:NTTA`.

Reply:
It's present in the `US:TX:Toll` block, this was a duplicate.  Is it not showing in the map?

Reply:
Yep, seems to be working here:
https://preview.ourmap.us/pr/1173/#map=12/32.7113/-97.36525